src/main.rs

```diff
@@ -38,9 +40,78 @@ async fn main() -> Result<()> {
     else {
         // use alternate logging for TUI
         telemetry::initialize_logs()?;
-        let state = state::initialize(cli.clone());
-        run_app(cli.clone(), state).await?;
+        let state = state::initialize(cli.config_path());
+        run_app(state).await?;
     }
 
     Ok(())
 }
+
+const LONG_ABOUT: &str = "
```

Comment:
doesnt need to be in this PR but maybe we put all of the clap logic in something like `args.rs`?

Reply:
Done in 5e88d0a